### PR TITLE
Sc 18

### PR DESCRIPTION
**Pull request by OleksandrRyzhov2299**
Made class EnumColoramaext, func coloramaText , .venv, imported colorama and added style

### CODE REVIEW

File: src/utils/colorama.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+from  colorama import Fore, Back , Style
+
+class EnumColoramaText(Enum):
+    SUCCESS = 1
+    WARNING = 2
+    ERROR = 3
+
+
+def coloramaText(text: str, colorama: EnumColoramaText) -> str:
```
Comment (anedost) [inline src/utils/colorama.py]: need rename 
like "responceWithColorama" etc.

File: src/utils/colorama.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    if colorama == EnumColoramaText.SUCCESS:
+        return Fore.GREEN + f"🔴 {text}"
```
Comment (anedost) [inline src/utils/colorama.py]: red circle

File: src/utils/colorama.py
Comment (anedost) [inline src/utils/colorama.py:1]: don't use global naming in files "colorama.py" its name in lib

File: src/utils/colorama.py (hunk 2 above)
Comment (OleksandrRyzhov2299) [reply src/utils/colorama.py]: Done

File: src/utils/colorama.py
Comment (OleksandrRyzhov2299) [reply src/utils/colorama.py:1]: chenged

File: src/utils/colorama.py (hunk 1 above)
Comment (OleksandrRyzhov2299) [reply src/utils/colorama.py]: chenged name func

**Review by yuratouch — APPROVED** [review]

**Pull request merged**